src/components/activity/Hompage.js

```diff
@@ -0,0 +1,9 @@
+function Homepage() {
+  return (
+    <div>
+      <h1>Hompage</h1>
+    </div>
```

Comment:
If you want to do this, please consider a `header` element to wrap the `h1` (if it fits your needs, of course).